README.md

```diff
@@ -175,7 +181,23 @@ vbrickEmbed.setVolume(0.5);
 }
 ```
 
-#### Removing Video
+##### Get Video Status
+You can get the status of the video using status variable. See example below
+
+`vbrickEmbed.playerStatus;`
+
+#### Player Status Values
+* Buffering
+* Ended
+* Initializing
+* Paused
+* Playing
+* Seeking
+
```

Comment:
Maybe also mention that the `playerStatusChanged` event reports when this value changes

Reply:
done